product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -4,9 +4,52 @@ redirects:
 - /biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/01_vpc_endpoint/
 ---
 
+
+
+AWS VPC endpoint (AWS Private Link) service is a network interface that securely connects a private IP address from your AWS VPC to an external service. You grant access only to a single cluster instead of the entire BigAnimal resource VPC, thus ensuring maximum network isolation. Other advantages include:
+
+-   You need to configure the PrivateLink only once. Then you can use multiple VPC endpoints to connect applications from different VPCs.
+-   There's no risk of IP address conflicts.
+
+There's an associated cost of resources, however.
+
+For more information, see [VPC endpoint services (AWS PrivateLink)](https://docs.aws.amazon.com/vpc/latest/privatelink/endpoint-service.html).
+
+The way you create a private endpoint differs when you are using your own AWS account as opposed to using BigAnimal's cloud account.
+
+## Using BigAnimal's cloud account
+
+When using BigAnimal's cloud account, you provide BigAnimal with your AWS account ID when creating a cluster. BigAnimal, in turn, provides you with a AWS service name, which you can use to connect to your cluster privately.
+
+1. When creating your cluster, on the **Cluster Settings** tab, in the **Network Connectivity** section:
+   1. Select **Private**.
+
+   1. Enter your application's AWS account ID.
```

Comment:
@valeriodelsarto @mattmanieri , are there any restrictions in the UI to which the customer will need to enter their AWS Account ID?  For example, 123456789012 vs 1234-5678-9012?

Reply:
Good question @jasonparton, I've suggested to use some basic UI validations in that field that takes for granted that the supported way to provide this value is the first one that you said, without dashes every 4 digits. You can see the regex that I've suggested to use [here](https://enterprisedb.atlassian.net/browse/UPM-19834). I think we can either write here an example value (like `123456789012` as you said) or explicitly say that the account ID needs to be wrote with only digits without dashes. WDYT?

Reply:
agree...think we'll encounter less issue if we simply accept the number-letter combination w/o dashes as a requirement.  Think it would also be good to show a simple example in the UI to avoid confusion.